components/engine_rocks/src/cf.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+use engine_traits::CFHandle;
+use rocksdb::CFHandle as RawCFHandle;
+use std::mem;
+
+// FIXME: This a nasty representation due to the lack of generic associated
+// types. See comment on the KvEngine::CFHandle associated type. This could also
+// be fixed if the CFHandle impl was defined inside the rust-rocksdb crate where
+// the RawCFHandles are managed, but that would be an ugly abstraction
+// violation.
+#[repr(transparent)]
+pub struct RocksCFHandle(RawCFHandle);
```

Comment:
em, seem rust-rocksdb has already provide a Rust wrapper, why do we need to use wrap Raw again?

/cc @overvenus @kennytm

Reply:
It needs a wrapper because this type implements a trait that rust-rocksdb doesn't have access to.